src/juliaexepath.ts

```diff
@@ -58,7 +59,12 @@ export async function getJuliaExePath() {
             }
         }
         else {
-            actualJuliaExePath = g_settings.juliaExePath;
+            if (g_settings.juliaExePath.includes(path.sep)) {
+                actualJuliaExePath = g_settings.juliaExePath;
+            } else {
+                // resolve full path
+                actualJuliaExePath = which.sync(g_settings.juliaExePath, {nothrow: true});
```

Comment:
Do you think we could use [this](http://2ality.com/2017/05/util-promisify.html#using-an-async-function) technique to actually use the async version here? I've tried pretty hard so far to not use sync versions of anything throughout the extension.

Reply:
Sure, I've updated the PR accordingly.